fluentui_persona/src/main/java/com/microsoft/fluentui/persona/PersonaChipView.kt

```diff
@@ -80,6 +78,17 @@ class PersonaChipView : TemplateView {
             field = value
             updateViews()
         }
+
+    var avatarContentDescriptionLabel: String = ""
+        set(value) {
+            if (field == value)
+                return
+            field = value
+            avatarView?.apply {
+                avatarContentDescriptionLabel = this@PersonaChipView.avatarContentDescriptionLabel
+                isFocusable = true
```

Comment:
(same for this case, should isFocusable = false if the value is null or empty?)

Reply:
makes sense

Reply:
thanks, if it makes sense!